Core/clim-core/gadgets.lisp

```diff
@@ -555,8 +555,9 @@ and must never be nil.")
 (defgeneric radio-box-selections (radio-box))
```

Comment:
In the first line of the commit message, please describe concise what the commit does.

`fixes #NNN` can be the final line of the commit message.

Reply:
this good?

Reply:
Yes. Though, we usually indicate the changed subsystem (using the term loosely here). For example: "gadgets: implement radio-box-selections …".

Can we ask you to tweak this once more and also squash the second commit into the first?

Reply:
like this?